addons/main/config.cpp

```diff
@@ -35,9 +33,11 @@ class CfgPatches {
             "cba_optics",
             "cba_disposable"
         };
-        versionDesc = "C.B.A.";
-        VERSION_CONFIG;
+        author = "$STR_CBA_Author";
         authors[] = {};
+        url = "$STR_CBA_URL";
+        VERSION_CONFIG;
+        versionDesc = "C.B.A.";
```

Comment:
```suggestion
        versionDesc = "$STR_CBA_Author";
```
This is weird atm.

Reply:
Agreed, remnant from CBA_A2. I would not put author there though, it's supposed to be a short description, ~~but I don't even know where it's used~~ apparently not used anywhere, just a dummy config entry.

Reply:
I don't think it is even read by anything. It is probably an entry from CBA_A2 that is superseded by the `name` entry from A3 base game.

Reply:
Yeah, sounds good, I'll delete.

Reply:
Also removed wiki section on that.